src/store/entriesSlice.ts

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+
+const initialState = {
+    entries: {}
```

Comment:
From what I've seen in the React tutorial I'm following along with, you might have to first make a single entry interface with all the properties that a single entry would have, and then you would say that entries is going to be an array of those entry interfaces. Maybe it would look something like:

interface Entry {
     <add properties here with data types>
};

const initialState = {
     entries: Entry[]=[];
};

I _think_ this is how you would do it?

Reply:
The synatx for initialState might be this instead:

const initialState = {
    entries: [] as Entry[]
};

Try both and see which one doesn't throw errors!